src/content/notInNavigation/infrastructure-release-notes.md

```diff
@@ -6,6 +6,32 @@ description: Chromatic's browser infrastructure release notes
 
 # Chromatic Capture Cloud release notes
 
+## Version 7
+
+Welcome to `Chromatic Capture Cloud version 7` released February 2025.
+
+**Status**: Beta ([opt in for upgrade](/docs/infrastructure-upgrades#opt-in-to-upgrade))
+
+Key highlights on this release:
+
+- **Handle clipped elements:** Previously Chromatic calculated the snapshot size without accounting for `overflow: clipped` elements, those are now clipped.
+- `Intl.ListFormat` now works in Safari thanks to the version upgrade.
+- **Improved video support:** Some video services, like BrightCove and Wistia, use ephemeral source URLs that cause encoding errors in snapshots when `autoplay` is enabled. Chromatic now handles these cases correctly by displaying a poster or a blank video if no poster exists.
+- **Better Redirect Detection:** Chromatic now alerts you when a story reloads the page, causing a snapshot error. Previously, these stories would fail silently. Now, you'll know what caused the error so you can fix the story.
```

Comment:
Being a bit nitpicky here but the stories would throw an error but mentions "missing helpers" and a stack trace which is all internal and is confusing to the user!

Reply:
Yep! If the page reloads, we lose all our helpers so we get a JS error on the internal stuff we expect to exist.

Reply:
how's this?

Reply:
Perfect! Thank you!